### ISSUE TITLE
Option --test break the results

### ISSUE THREAD

**Issue by elMor3no** [eikenb/terminal-colors#6]
Hi 
Im doing the packaging to colortest-python but testing the program im get this issue

$ ./terminal-colors --test
Running tests...
**********************************************************************
File "./terminal-colors", line 187, in __main__.term16.ansicodes
Failed example:
    term16.ansicodes()[1][:6]
Expected:
    ['      ', '   _   ', '  40m  ', '  41m  ', '  42m  ', '  43m  ']
Got:
    ['      ', '  _  ', ' 40m ', ' 41m ', ' 42m ', ' 43m ']
**********************************************************************
File "./terminal-colors", line 191, in __main__.term16.ansicodes
Failed example:
    term16.ansicodes()[2][17][0]
Expected:
    '\x1b[1;37m\x1b[_  gYw  \x1b[0m'
Got:
    '\x1b[1;37m\x1b[_ gYw \x1b[0m'
**********************************************************************
1 items had failures:
   2 of   4 in __main__.term16.ansicodes
***Test Failed*** 2 failures.
Done.

**Comment by eikenb** [comment]
Thanks for the report. I'll look into that.

**Comment by eikenb** [comment]
Some tests (such as they are) were out of date. Just pushed up the fix and bumped the version to 3.0.1.

When I wrote it I was mainly interested in testing out the class/namespacing thing and wasn't worried about testing. After the fact I (when adding tests) I saw that I'd need a pretty big refactor to test things properly and decided against it. I remember it as one of the many places that reinforced my new rule of always writing tests along with the code. :)
